### ISSUE TITLE
Contribution guidelines

### ISSUE THREAD

**Issue by benjamin-work** [skorch-dev/skorch#229]
We should add guidelines on how to contribute.

**Comment by damienlancry** [comment]
Hi are there guidelines to contribute somewhere now?

**Comment by BenjaminBossan** [comment]
No, but if you find something nice, you can make a proposal. Probably something about being reasonable and being nice :)
